tests/integration_frameworks/conftest.py

```diff
@@ -55,10 +67,18 @@ def test_client(
     test_id: str,
     worker_id: str,
     test_agent: TestAgentAPI,
+    framework_app_name: str,
 ) -> Generator[FrameworkTestClientApi, None, None]:
+    framework_scenario: IntegrationFrameworksScenario | None = getattr(
+        scenarios, f"integration_frameworks_{framework_app_name}"
+    )
+
+    if framework_scenario is None:
+        pytest.exit(f"Framework scenario for {framework_app_name} not found", 1)
+
     context.scenario.parametrized_tests_metadata[request.node.nodeid] = dict(library_env)
 
-    with scenarios.integration_frameworks.get_client(
+    with framework_scenario.get_client(  # type: ignore[union-attr]
```

Comment:
this is the main change i'm unsure about. with the two different scenarios now (and in the future, `N` scenarios for the number of auto-instrumentations we want to test), this was the way we were thinking about selecting the correct integration frameworks scenario. otherwise, we weren't too sure about the best approach to handling this, but open to suggestions!